paper.tex

```diff
@@ -337,9 +357,14 @@ \subsection{Scenario Description}
 
 The described scenario is used as closely as possible in each of the four
 cases.  However, a few small case-dependent changes to the scenarios relating to
-reactor configuration are necessary.  For the cases using quarterly time steps
-with individual reactors, cycle and outage times must be adjusted to be a
-multiple of 3 months.  In order to make the comparison more valid, the cases
+reactor configuration are necessary.  For case \emph{QI}, 
+cycle and outage times must be adjusted to be a
+multiple of 3 months. 
+%% this tweak is a distraction.  Why not simply declare the SFR cycle length &
+%% outage as 12/3 from the beginning.  Is this because the fuel recipes you
+%% are using assume the other parameters and you don't want to sweep that
+%% under the rug?
```

Comment:
It makes more sense in the context of my dissertation - the exp2 scenarios are tweaked w.r.t. the experiment 1 scenarios.  And also, this tweak is a divergence from the FCO work - which arguably isn't a big deal.  We can tweak away this tweak discussion.